orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CreateServerGroupTaskSpec.groovy

```diff
@@ -93,6 +99,57 @@ class CreateServerGroupTaskSpec extends Specification {
     "cCloud"      | [["cOp": "baz"]] || baseOutput + ["kato.result.expected": true]
   }
 
+  @Unroll
+  def "should fail to create server group"() {
+    given:
+    KatoService katoService = Mock(KatoService)
+    def task = new CreateServerGroupTask(kato: katoService, serverGroupCreators: [awsCreator])
+    def stage = ExecutionBuilder.stage {
+      type = "whatever"
+      context["credentials"] = "abc"
+      context["cloudProvider"] = cloudProvider
+      context["moniker"] = ["app": "applicationundertest", "stack": "integrationenvironment", "detail": "applicationdetailsundertest"]
+    }
+
+    when:
+    def result = task.execute(stage)
+
+    then:
+    thrown IllegalStateException
+
+    where:
+    cloudProvider | ops              || outputs
```

Comment:
it's convention to line all the `|`s up so it makes a nice table

Reply:
thanks, fixed the format